src/storage/index.ts

```diff
@@ -387,46 +386,54 @@ function handleSignedEncryptedContents(caller: UserSession, path: string, stored
   })
 }
 
+export interface GetFileUrlOptions {
+  /**
+   * The Blockstack ID to lookup for multi-player storage. 
+   * If not specified, the currently signed in username is used.
+   */
+  username?: string;
+  /**
+   * The app to lookup for multi-player storage - defaults to current origin. 
+   * @default `window.location.origin` 
+   * Only if available in the executing environment, otherwise `undefined`.
+   */
+  app?: string;
+  /**
+   * The URL to use for zonefile lookup. If falsey, this will use 
```

Comment:
```  
 The URL to use for zonefile lookup. If falsey, this uses
    the blockstack.js's [[getNameInfo]] function instead. 
```

Check that link to the function ...I didn't see it exposed in your artifact.

Reply:
I don't think `getNameInfo` is exposed anywhere. It's a method on the `BlockstackNetwork` class instance. Some methods on that class are wrapped and exposed but not this one. Should we expose the function, or remove the reference to it in the docs?

Reply:
Also the `[[getNameInfo]]` symbol reference was not added or broken in this PR. Its existed (in another location), and has been broken.

Reply:
This functions actually seems very useful IMO. I wrapped and exposed it for easy public use, and got the links fixed.